.npmignore

```diff
@@ -0,0 +1,3 @@
+node_modules/
+app/
```

Comment:
I don't think we need app/ here (we should probably also remove it from the gitignore since it doesn't exist anymore?)